agent/lib/console_server.js

```diff
@@ -197,11 +214,15 @@ ConsoleServer.prototype.listen = function (env, callback) {
         } else if (request.method === 'GET') {
             try {
                 var user = myutils.basic_auth(request);
-                var transform;
+                var next;
                 if (url.parse(request.url).pathname === '/help.html' && env.MESSAGING_ROUTE_HOSTNAME !== undefined) {
```

Comment:
This is not set anywhere is it?

Reply:
Yes, it is. It is set on startup based on the hostname in the route.